From 4efa13b3f0f4ddab6fa6205b09ce7784d1d5f3a7 Mon Sep 17 00:00:00 2001
From: Patrick Gaskin <patrick@pgaskin.net>
Date: Sun, 1 Oct 2023 18:05:15 -0400
Subject: [PATCH 2/9] Improve floating toolbar usability and appearance

Updating:
- Update the resources.
- Find the method setting the opacity of an ImageButton (i.e., the FAB) (try: ag '0x3f800000' -C5 | ag '0x3f000000' | ag 'const/high16')
- Test to ensure the toolbar is translucent white.
- Test to ensure the toolbar and FAB are larger.

diff --git a/res/values/colors.xml b/res/values/colors.xml
index a35406b..a9090b1 100644
--- a/res/values/colors.xml
+++ b/res/values/colors.xml
@@ -91,7 +91,7 @@
     <color name="foreground_material_dark">@android:color/white</color>
     <color name="foreground_material_light">@android:color/black</color>
     <color name="fragment_background">#ffeeeeee</color>
-    <color name="fullscreen_toolbar_bg">#ff000000</color>
+    <color name="fullscreen_toolbar_bg">#aaffffff</color>
     <color name="fullscreen_toolbar_blue">#ff0036d8</color>
     <color name="highlighted_text_material_dark">#6680cbc4</color>
     <color name="highlighted_text_material_light">#66009688</color>
diff --git a/res/values/dimens.xml b/res/values/dimens.xml
index 30b3ce1..3d825e7 100644
--- a/res/values/dimens.xml
+++ b/res/values/dimens.xml
@@ -883,7 +883,7 @@
     <dimen name="thumbnail_height">72.0dip</dimen>
     <dimen name="thumbnail_width">96.0dip</dimen>
     <dimen name="toolbar_dropshadow_height">4.0dip</dimen>
-    <dimen name="toolbar_fab_layout_size">54.0dip</dimen>
+    <dimen name="toolbar_fab_layout_size">96.0dip</dimen>
     <item type="dimen" name="toolbar_layout_height">@dimen/toolbar_fab_layout_size</item>
     <dimen name="toolbar_layout_width">260.0dip</dimen>
     <dimen name="toolbar_topshadow_height">2.0dip</dimen>
diff --git a/smali/c3/g.smali b/smali/c3/g.smali
index bc3c111..64ea846 100644
--- a/smali/c3/g.smali
+++ b/smali/c3/g.smali
@@ -202,7 +202,7 @@
 
     const/high16 v1, 0x3f800000    # 1.0f
 
-    const/high16 v3, 0x3f000000    # 0.5f
+    const/high16 v3, 0x3f400000    # 0.75f
 
     if-eqz v0, :cond_3
 
